### PR TITLE
Fix tensorflow_serving_container.ipynb

### PR DESCRIPTION
**Pull request by hsl89**
Link to the notebook:
https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb

### CODE REVIEW

File: sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb [hunk 1]
```diff
@@ -1,47 +1,90 @@
 {
```
Comment (jkroll-aws) [inline sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb:1]: <p>Do you want to keep all the outputs in this notebook? And all the warnings/info outputs later on? It doesn't seem like they're adding key information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by jkroll-aws — DISMISSED** [review]

**Review by jkroll-aws — APPROVED** [review]

File: sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb (hunk 1 above)
Comment (hovanesgasparian) [reply sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb:1]: I would beg to disagree... I would have loved to see what might go wrong, as I'm getting plenty of warnings and errors myself...

Comment (jkroll-aws) [reply sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb:1]: @hovanesgasparian You can still view the previous commits with the outputs. Linked below. Please open an issue if you encounter other errors and we will attempt to replicate and fix.
https://app.reviewnb.com/aws/amazon-sagemaker-examples/commit/d20140ed1eb81691100607e3a8711ddd65d03681/
https://app.reviewnb.com/aws/amazon-sagemaker-examples/commit/950900ff773049dc2b0caa80e1377f5a41671172/
https://app.reviewnb.com/aws/amazon-sagemaker-examples/commit/bf197a9d46d0b2799472b4cccebddded5188ce7e/

Comment (hovanesgasparian) [reply sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb:1]: Oh awesome, thank you! I tried that before, and got the Jedi Octocat 404 page, so I thought I just didn't have permission!

**Pull request merged**